lib/Doctrine/Common/Cache/SQLite3Cache.php

```diff
@@ -165,6 +165,10 @@ private function findById($id, bool $includeData = true) : ?array
             $idField
         ));
 
+        if ($statement === false) {
+            throw new \RuntimeException('SQLite internal database structure is broken: ' . (error_get_last()['message'] ?? ''));
```

Comment:
> SQLite internal database structure is broken

You don't know that for sure, there might be other reasons for `prepare` returning `false`, right?

Reply:
I think there may be more reasons and I don't know exactly how to find them. This will most likely be caused by a broken file.

The ʻerror_get_last` function should return information about the last error that occurred, for example, when trying to read a non-existent file to which there were no rights.

Reply:
> I think there may be more reasons and I don't know exactly how to find them. 

Then maybe don't be so specific, or leave room for other explanations in the message. Also, it seems weird that you get `false` and not an exception. Are you using `doctrine/dbal`? It should configure PDO to throw on error: https://github.com/doctrine/dbal/search?q=ATTR_ERRMODE